FIX: Incomplete datalad push information

oesteban opened the pull request:
These changes substantially improve the section about adding more data into the DataLad dataset.

In addition to that, it revises a few related aspects about physio and eyetracking.

Resolves: #228.
Relates: #235.
Relates: #185.

oesteban committed:
fix: incomplete datalad push information

These changes substantially improve the section about adding more data
into the DataLad dataset.

In addition to that, it revises a few related aspects about physio and
eyetracking.

Resolves: #228.
Relates: #235.
Relates: #185.

oesteban commented:
I failed to mention #74. This PR opens a new section in the SOPs because we had a section for physio and ET, but it only covered physio.

I have added a section for ET, but it must be filled in with the procedure (I see that is part of #74).

esavary commented on docs/data-management/post-session.md:
````diff
@@ -163,7 +185,30 @@ As new sessions are collected, the corresponding BIDS structures MUST be saved w
     datalad push --to origin
     ```
+
+    !!! danger "Always double-check that data in the annex are uploaded to the RIA store"
````
I think it would be useful to add the command below.

oesteban replied:
what do you mean?

esavary replied:
Is there a method to double check?

oesteban replied:
AFAIK you can clone the dataset somewhere else and test it pulls down the data :)

Also, when doing the push to ria-storage explicitly it should tell you if it failed.

oesteban commented:
> > I'm currently hitting [mmatl/pyrender#13](https://github.com/mmatl/pyrender/issues/13) and I believe this [psychopy/psychopy#5407](https://github.com/psychopy/psychopy/issues/5407) could be one cause.
> 
> I'm not sure I understood your last comment. Is it possibly related to another pull request?

totally, sorry about that.

oesteban commented:
@esavary I've deleted the comments following mine, which you correctly identified as completely unrelated to the conversation. I've deleted them to reduce the noise on the mentioned repositories.
